cpp/velox/compute/VeloxExecutionCtx.h

```diff
@@ -142,6 +143,8 @@ class VeloxExecutionCtx final : public ExecutionCtx {
   ResourceMap<std::shared_ptr<ResultIterator>> resultIteratorHolder_;
 
   std::shared_ptr<const facebook::velox::core::PlanNode> veloxPlan_;
+
+  std::unordered_map<int32_t, ResourceHandle> emptySchemaBatchLoopUp_;
```

Comment:
can you point out to me, where do we release the resource ?

Reply:
Batches should be released during destructing member `columnarBatchHolder_` from a JVM side call to release `ExecutionCtx` when task ends.

Reply:
I did not see where release `emptySchemaBatchLoopUp_`.. do you mean it will be released when calling `delete executionCtx` ? it seems we did not override `~ExecutionCtx()`

Reply:
> I did not see where release `emptySchemaBatchLoopUp_`.. do you mean it will be released when calling `delete executionCtx` ? it seems we did not override `~ExecutionCtx()`

You are right, we don't have to override dtor since the default dtor can handle it automatically.